h2o-core/src/main/java/hex/ModelMetricsRegression.java

```diff
@@ -19,13 +19,18 @@ public class ModelMetricsRegression extends ModelMetricsSupervised {
   public double residual_deviance() { return _mean_residual_deviance; }
   @SuppressWarnings("unused")
   public double mean_residual_deviance() { return _mean_residual_deviance; }
+  public final double _mean_residual_deviance_2;
+  public double residual_deviance_2() { return _mean_residual_deviance_2; }
+  @SuppressWarnings("unused")
+  public double mean_residual_deviance_2() { return _mean_residual_deviance_2; }
   public final double _mean_absolute_error;
   public double mae() { return _mean_absolute_error; }
   public final double _root_mean_squared_log_error;
   public double rmsle() { return _root_mean_squared_log_error; }
-  public ModelMetricsRegression(Model model, Frame frame, long nobs, double mse, double sigma, double mae,double rmsle, double meanResidualDeviance, CustomMetric customMetric) {
+  public ModelMetricsRegression(Model model, Frame frame, long nobs, double mse, double sigma, double mae,double rmsle, double meanResidualDeviance, double meanResidualDeviance2, CustomMetric customMetric) {
```

Comment:
Can you keep the original ModelMetricsRegression like this:

public ModelMetricsRegression(Model model, Frame frame, long nobs, double mse, double sigma, double mae,double rmsle, double meanResidualDeviance, CustomMetric customMetric) {
return ModelMetricsRegression( model,  frame,  nobs, mse,  sigma, mae, rmsle, double meanResidualDeviance, Double.Na, customMetric);

Reply:
Oh thanks for a hint! Done.